paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -492,6 +477,12 @@ def autoscale_marathon_instance(marathon_service_config, marathon_tasks, mesos_t
 
     safe_downscaling_threshold = int(current_instances * 0.7)
     if new_instance_count != current_instances:
+        if new_instance_count < current_instances and task_data_insufficient:
+            write_to_log(config=marathon_service_config,
+                         line='Delaying scaling *down* as we found too many or too few tasks running in marathon. '
```

Comment:
Delaying scaling *down* as we found too few healthy tasks running in marathon. '